tests/posix/common/src/timer.c

```diff
@@ -122,6 +122,29 @@ ZTEST(timer, test_timer_overrun)
 	zassert_equal(timer_getoverrun(timerid), 4, "Number of overruns is incorrect");
 }
 
+ZTEST(timer, test_one_shot__SIGEV_SIGNAL)
+{
+	struct sigevent sig = {0};
```

Comment:
This line of code should only be added in the second commit. There is no need to change it as well in the third commit.

Reply:
Sorry for this. Commit for formatting existing code should only change existing code now.
Concerning https://github.com/zephyrproject-rtos/zephyr/pull/80687#pullrequestreview-2413734406, not sure to get all what you mean by 

> edit the one line so that it reads as {0}